common/src/main/java/com/wynntils/models/containers/BankModel.java

```diff
@@ -20,6 +20,7 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 public class BankModel extends Model {
+    // When storage supports upfixing, change to finalAccountBankPage
```

Comment:
We will not be able to support this, before 2.0.4, so might as well remove the comment here and below.

Unless @magicus or me adds a system for it? I don't think it is that far, but I wouldn't want to interfere with magicus' config refactors.

Reply:
Oh.. I see that you kept the variable name.. My bad. I guess keep the comments, and this is much less of a priority.

Reply:
Yeah not a priority just since I just want it to be clear what is account vs character bank now